### ISSUE TITLE
batchnormalization deprecated

### ISSUE THREAD

**Issue by mfumagalli** [ulasisik/balancing-selection#3]
in Model.py
from keras.layers.normalization import BatchNormalization
should be replaced with
from tensorflow.keras.layers import BatchNormalization
